Tagesspiegel.js

```diff
@@ -97,17 +98,34 @@ function scrape(doc, url) {
 	}
 	
 	newItem.url = ZU.xpathText(doc, '//link[@rel="canonical"]/@href') || url;
-	// Printurl (add "v_print," before the article ID and "?p=" at the end) 
-	var printurl = newItem.url.replace(/^(.*\/)(\d+.html$)/, '$1v_print,$2?p=');
 	newItem.attachments.push({
-		url: printurl,
+		url: newItem.url,
 		title: "Snapshot",
 		mimeType: "text/html"
 	}); 
 	
 	// Tags
-	var tags = ZU.xpathText(doc, "//meta[@name='news_keywords']/@content");
-	if (tags) var tags= tags.split(","); // this seems to work even if there's no |
+	/* We read the tags from the initialisation of variable cmsObject.
+	 * This object and the keywords are defined in a head script tag.
+	 */
+	let tags = [];
+	let scriptItems = doc.querySelectorAll('head > script');
+	if (scriptItems) {
+		for (let i = 0; i < scriptItems.length; i++) {
+			let scriptItemText = scriptItems[i].textContent;
+			// search for script tag that declares the variable
+			if (!scriptItemText.match(/var\scmsObject\s=/)) {
```

Comment:
Maybe try this:

```suggestion
			if (!scriptItemText.match(/var\s+cmsObject\s*=/)) {
```

So that we'll be OK if the script's formatting changes.